### ISSUE TITLE
E2E: Bump ec2 retention time to 8 hours

### ISSUE THREAD

**Issue by mrajashree** [aws/eks-anywhere#269]
Currently the ec2 instances for failed jobs get deleted after 1-1.5 hours. Bump this limit so the instance and artifacts on it are available for debugging purposes for longer

**Comment by TerryHowe** [comment]
Can't we just rerun tests to look at failures? 8 hours seems like a long time.

**Comment by mrajashree** [comment]
@TerryHowe reruns won't necessarily lead to same results, right now anyone debugging e2e test failures needs to get on the instance within that 1.5 hr window which seems less. I've created [this task](https://github.com/aws/eks-anywhere/issues/271) to get and upload all required artifacts/logs from the instance to s3 so we don't need the instance for that long. But still for docker e2e tests since the containers get created on the same ec2 instance we might have to keep it for longer. I guess the ec2 instance limit has also been increased for that account? (@vignesh-goutham can you confirm this)

**Comment by vignesh-goutham** [comment]
Thats right @mrajashree. I increased the ec2 instance limits. We should not be having any issues in terms of limits for now.

**Comment by mitalipaygude** [comment]
On it!
